src/utils/handleNFTImages.ts

```diff
@@ -36,24 +38,25 @@ export function handleNFTImages({
 
   const isSVG = mimeType === MimeType.SVG;
   const isGIF = mimeType === MimeType.GIF;
-
-  const cardSize = Math.floor((Math.ceil(CardSize) * pixelRatio) / 3);
-
   const highResUrl =
     (isSVG
       ? // don't sign if SVG, will be handled by UniqueTokenImage component
         originalUrl
-      : maybeSignUri(url, {
-          // decrease size for GIFs to avoid hitting imgix's 10MB limit
-          w: isGIF ? cardSize : FULL_NFT_IMAGE_SIZE,
-        })) ?? null;
+      : url?.startsWith?.(GOOGLE_USER_CONTENT_URL)
+        ? url.replace(/=s\d+$/, `?s=${FULL_NFT_IMAGE_SIZE}`)
```

Comment:
i cant remember the context exactly, but i think i removed the google url stuff bc urls that did not end in `?s={size}` were not getting resized

Reply:
good call out ill add a case for that

Reply:
looks like out initial logic was wrong at some point to and we were not resizing the images properly, google may have changed the param structure

im unable to find a url that doesnt have on a few wallets with my upcoming change so we should be good